### PR TITLE
docs: meeting notes 2020-09-30

### PR DESCRIPTION
**Pull request by leogr**
Signed-off-by: Leonardo Grasso <me@leonardograsso.com>

TODO: recording link
/hold

### CODE REVIEW

**Review by fntlnz — DISMISSED** [review]

File: meeting-notes/2020-09-30.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Falco Community Call
+
+[More Information](https://github.com/falcosecurity/community)
+[Zoom Link](https://zoom.us/my/cncffalcoproject)
+[Recording](https://)
```
Comment (leogr) [inline meeting-notes/2020-09-30.md:5]:
```suggestion
[Recording](https://youtu.be/mDEsExkpgLw)
```

Comment (leogr) [inline meeting-notes/2020-09-30.md:5]:
```suggestion
[Recording](https://youtu.be/mDEsExkpgLw)
```

**Review by leodido — APPROVED** [review]

**Review by fntlnz — APPROVED** [review]

File: meeting-notes/2020-09-30.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- Thomas Labarussias ([Issif](https://github.com/Issif))
+- Spencer Krum ([nibalizer](https://github.com/nibalizer))
+
+### This week highlights
+
+- Falco 0.26.0 is out (?)
+    - [Release on GitHub](https://github.com/falcosecurity/falco/releases) is still draft
+    - [bug](https://github.com/falcosecurity/falco/issues/1418) in the default ruleset for 0.26.0 - so probably we need to release a 0.26.1 hotfix release
+    - We decided to publish the 0.26.0 live on the call and we will skip the announcement since we have to release 0.26.1 with the fix. We will do announcement and blog post all together for 0.26.1.
+    - Lorenzo and Leogr to release 0.26.1
```
Comment (leodido) [inline meeting-notes/2020-09-30.md:26]: As decided internally, I'm going to release 0.26.1 with Lorenzo.
Doing it right now!

File: meeting-notes/2020-09-30.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+### Agenda
+
+- [Lore/Leo/Leogr/Loris] Release window
```
Comment (leodido) [inline meeting-notes/2020-09-30.md:30]: 🥳

**Pull request merged**